### PR TITLE
Added FAQ line for issue 'Missing root span' in trace details.

### PR DESCRIPTION
**Pull request by hhovsepy**
Fix for issue https://github.com/kiali/kiali/issues/4413

### CODE REVIEW

File: content/en/docs/FAQ/distributed-tracing.md [hunk 1]
```diff
@@ -83,3 +83,10 @@ When configured, this URL will be used to generate a couple of links to Jaeger w
 On the Application detail page, the Traces tab might redirect to Jaeger via an external link instead of showing the Kiali Tracing view. It happens when you have the `url` field configured, but not `in_cluster_url`, which means the Kiali backend will not be able to connect to Jaeger.
 
 To fix it, configure `in_cluster_url` in [Kiali CR](https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml) or ConfigMap.
+
+
+### Why I see "Missing root span" for the root span of some span details on Traces tab?
```
Comment (jmazzitelli) [inline content/en/docs/FAQ/distributed-tracing.md]:
```suggestion
### Why do I see "Missing root span" for the root span of some span details on Traces tab?
```

File: content/en/docs/FAQ/distributed-tracing.md [hunk 2]
```diff
@@ -83,3 +83,10 @@ When configured, this URL will be used to generate a couple of links to Jaeger w
...
+
+![Missing root span](/images/documentation/faq/tracing/missing-root-span.png)
+
+In Traces tab, while clicking on a trace, it shows the details of that trace and information about spans. This details also includes the root span information. But for the traces the traffic of which is not comming from ingress-gateway, the root span information is not available in Jaeger, thus Kiali is displaying "Missing root span" for those traces details and tooltips in Traces tab and in Graph pages.
```
Comment (jmazzitelli) [inline content/en/docs/FAQ/distributed-tracing.md]:
```suggestion
In Traces tab, while clicking on a trace, it shows the details of that trace and information about spans. These details also include the root span information. But for the traces for traffic that is not coming from ingress-gateway, the root span information is not available in Jaeger, thus Kiali is displaying "Missing root span" for those traces' details and tooltips in Traces tab and in Graph pages.
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
minor corrections requested

**Comment by hhovsepy** [comment]
thank you, changes approached

**Comment by jmazzitelli** [comment]
@hhovsepy FYI for the future: do not squash-force-push commits after committing changes requested by reviewers. It makes it harder to re-review the PR. We should only have to review the parts that changed since the last review. But because of the squash, there is only a single commit and no one can see what you changed since the last commit when the PR was last reviewed.

**Review by jmazzitelli — APPROVED** [review]
lgtm

**Comment by hhovsepy** [comment]
> @hhovsepy FYI for the future: do not squash-force-push commits after committing changes requested by reviewers. It makes it harder to re-review the PR. We should only have to review the parts that changed since the last review. But because of the squash, there is only a single commit and no one can see what you changed since the last commit when the PR was last reviewed.

so squash commits after the review is finished, or leave them separately (not a good style) ?

**Comment by jmazzitelli** [comment]
> so squash commits after the review is finished, or leave them separately (not a good style) ?

During the PR review, just keep the individual commits (do not squash them) - as you continue to work on the PR and submit suggested changes, you should have multiple commits in the PR so people can review the changes without having to review the entire thing again (we can just review the last commit you did - since those will only show the diff since the last review).

AFTER everything is done, and you are ready to merge, THEN you click the "squash-merge" button in the github PR which will then squash commit everything into a single commit into the master branch.

So you only have multiple commits in your PR branch, but when they get merged into the master branch on the main repo, you end up with a single commit. The multiple commits are only temporary and only in your PR branch in your repo.

**Pull request merged**